core/src/main/scala/cromwell/core/WorkflowOptions.scala

```diff
@@ -56,19 +56,21 @@ object WorkflowOptions {
   case object FinalCallLogsDir extends WorkflowOption("final_call_logs_dir")
   case object FinalWorkflowOutputsDir extends WorkflowOption("final_workflow_outputs_dir")
   case object UseRelativeOutputPaths extends WorkflowOption(name = "use_relative_output_paths")
-  case object FinalWorkflowOutputsMode extends WorkflowOption("final_workflow_outputs_mode") {
-    // Default to Copy because that was originally the only behavior
-    def fromString(s: Option[String]): FinalWorkflowOutputsMode =
+  case object FinalWorkflowOutputsDirMetadata extends WorkflowOption("final_workflow_outputs_dir_metadata") {
```

Comment:
As long as it's well-documented (which it looks like it is) I think this is OK as is, but agree that the naming is awkward. Might be easier to name if it were a boolean, something like `FinalWorkflowOutputsDirStoredInOutputs`? 😬